xarray/convert.py

```diff
@@ -49,3 +62,142 @@ def set_cdms2_attrs(var, attrs):
     cdms2_var = cdms2.createVariable(var.values, axes=axes, id=dataarray.name)
     set_cdms2_attrs(cdms2_var, var.attrs)
     return cdms2_var
+
+
+# TODO: Add converting bounds from xarray to Iris and back
+def to_iris(dataarray):
+    """Convert a DataArray into a Iris Cube
+    """
+    # Iris not a hard dependency
+    import iris
+    # iris.unit is deprecated in Iris v1.9
+    import cf_units
+
+    def check_attrs(attrs, keys):
+        return dict((k, v) for k, v in attrs.items() if k in keys)
+
+    def get_args(attrs):
+        _args = {'attributes': filter_attrs(attrs, iris_forbidden_keys)}
+        _args.update(check_attrs(attrs, ('standard_name', 'long_name',)))
+        _unit_args = check_attrs(coord.attrs, ('calendar',))
+        if 'units' in attrs:
+            _args['units'] = cf_units.Unit(attrs['units'], **_unit_args)
+        return _args
+
+    def get_cell_methods(cell_methods_str):
+        """Converts string to iris cell method objects"""
+        cell_methods = []
+        _cell_method_words = [w.strip() for w in cell_methods_str.split(':')]
+        cm = {'coords': [], 'method': '', 'interval': [], 'comment': []}
+        skip = False
+        for i, word in enumerate(_cell_method_words):
+            # If this value is a comment or an interval don't read
+            if skip:
+                skip = False
+                continue
+            # If this word is an axis
+            if word not in cell_methods_strings | set(['interval', 'comment']):
```

Comment:
Is there something like `iris.coords.CellMethod.from_string` which handles this logic? It might be a happier path to expose such a method from Iris (which already is using this sort of logic internally) rather than incorporating it into xarray.